core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -98,7 +98,7 @@ <h4 class="bold-title">
                         </section>
                         <section class="runtime-option">
                             <div class="radio-container">
```

Comment:
Include 'disable-event' class here to include css same as C#

Reply:
Done